src/common/playlists/PlayList.jsx

```diff
@@ -58,7 +57,7 @@ const PlayList = () => {
     return <Loader />;
   }
 
-  if (plays?.length === 0) {
+  if (plays.length === 0) {
```

Comment:
Can you help me understand why did you remove optional chaining (`?`) here? I am confused.